source/dev_testing.md

```diff
@@ -29,6 +29,10 @@ e.g. in the folder `./odahu-flow/virtual_environment/` and activate one.
     ```bash 
     /odahu-flow$ odahuflowctl
     ```
+1. Also, you should have installed jq and rclone packages. if not:
+    ```bash 
+   $ sudo apt-get install jq rclone
```

Comment:
Once we have an `Unix-like` OS in previous requirements, I think that this string is excessive. We could just provide links like
```
you should have installed [jq ](https://stedolan.github.io/jq/) and [rclone](https://rclone.org/downloads/)
```

Reply:
I leave as it is because for user it easier just copy and paste.
And if they would like to find more information about it, they can do it but it's not so frequent action.

Reply:
@maxkochubey 
Or have you proposed it 'cause, for example, on MacOS we don't just have apt-get command?

Reply:
Yes, I mean, Fedora or CentOS users will be a little bit confused of this thing.